packages/gasket-nextjs/package.json

```diff
@@ -42,20 +42,18 @@
   },
   "homepage": "https://github.com/godaddy/gasket/tree/main/packages/gasket-nextjs",
   "dependencies": {
-    "@gasket/data": "^6.36.1",
+    "@gasket/data": "^6.34.6",
```

Comment:
Is this change intentional? The version should match up to what's current in the monorepo.